x-pack/apm-server/aggregation/spanmetrics/aggregator_test.go

```diff
@@ -251,16 +247,20 @@ func TestAggregateTransactionDroppedSpansStats(t *testing.T) {
 					Subtype:                    "elasticsearch",
 					DestinationServiceResource: "https://elasticsearch:9200",
 					Outcome:                    "success",
-					Count:                      &esDroppedCount,
-					DurationSumUs:              &esDurationSum,
+					Duration: model.AggregatedDuration{
+						Count: 10,
+						Sum:   1_500_000,
```

Comment:
```suggestion
						Sum:   1500 * time.Microsecond,
```

YMMV, but I find it easier to understand if the units are specified.

Reply:
ah yes, I partially fixed it on the assertion but forgot to do it here, thanks!